go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -444,11 +444,10 @@ func handleNetdevResources(resourceName string) (string, error) {
 	return netdevice, nil
 }
 
-func exportManagementPortAnnotation(link netlink.Link, nodeAnnotator kube.Annotator) error {
```

Comment:
Could you squash this commit with the previous commit? The reason is because the previous commit would be buggy on it's own without this commit right?

Reply:
No, the final commit is a refactor.